test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -382,6 +382,43 @@ def test_phase_vocoder_shape(self, rate, test_pseudo_complex):
         output_shape = (torch.view_as_complex(spec_stretch) if test_pseudo_complex else spec_stretch).shape
         assert output_shape == expected_shape
 
+    @parameterized.expand(
+        [
+            # words
+            ["", "", 0], # equal
+            ["abc", "abc", 0],
+            ["ᑌᑎIᑕO", "ᑌᑎIᑕO", 0],
+
+            ["abc", "", 3], # deletion
+            ["aa", "aaa", 1],
+            ["aaa", "aa", 1],
+            ["ᑌᑎI", "ᑌᑎIᑕO", 2],
+
+            ["aaa", "aba", 1], # substitution
+            ["aba", "aaa", 1],
+            ["aba", "   ", 3],
+
+            ["abc", "bcd", 2], # mix deletion and substitution
+            ["0ᑌᑎI", "ᑌᑎIᑕO", 3],
+
+            # sentences
+            [["hello", "", "Tᕮ᙭T"], ["hello", "", "Tᕮ᙭T"], 0], # equal
+
+            [["hello", "world"], ["hello", "world", "!"], 1], # deletion
+            [["hello", "world"], ["world"], 1],
+
+            [["Tᕮ᙭T", ], ["world"], 1], # substitution
+            [["Tᕮ᙭T", "XD"], ["world", "hello"], 2],
+            [["", "XD"], ["world", ""], 2],
+            ["aba", "   ", 3],
+
+            [["hello", "world"], ["world", "hello", "!"], 2], # mix deletion and substitution
+            [["Tᕮ᙭T", "world", "LOL", "XD"], ["world", "hello", "ʕ•́ᴥ•̀ʔっ"], 3],
+        ]
+    )
+    def test_simple_case_edit_distance(self, ref, hyp, distance):
+        assert F.edit_distance(ref, hyp) == distance
```

Comment:
Some cases are tested for flipped version but some are not. Can you check the flipped version as well?
Also can you rename `ref`/`hyp` to `seq1`/`seq2`?

```python
assert F.edit_distance(seq1, seq2) == distance
assert F.edit_distance(seq2, seq1) == distance
```

Reply:
Thanks for the reminder. I've fixed these [here](https://github.com/pytorch/audio/pull/1601/files#diff-cf61df2ded7de3af992d120c6b1530bddb6333ad62e825ce4c9251dafe9afd1dR421).